Lab2.ipynb

```diff
@@ -0,0 +1,677 @@
+      "text/plain": [
+       "<Figure size 1080x360 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "### Problem 1\n",
+    "# Part 1A\n",
+    "x0 = np.linspace(0, 10, 11)\n",
```

Comment:
In Problem 1, you aren't using nearly a large enough range for x0. As you can see on your plot, a Poisson distribution with mean 5 still has a decent probability at 10, so by truncating your distribution there you're missing a substantial amount of probability (about 1.5%). I would go out to at least 20 for accurate results. This doesn't matter much for your graphs but it has a substantial impact on the calculation in 1D.